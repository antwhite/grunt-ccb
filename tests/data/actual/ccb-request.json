{"headers":{"content-type":"application/json","accept":"application/json","host":"localhost:8888","authorization":"Basic dXNlcjpwYXNzd29yZA==","content-length":"253","connection":"keep-alive"},"url":"/issue/","body":{"fields":{"project":{"id":"4321"},"summary":"Deploying projectname projectversion to production [grunt-ccb_1234]","issuetype":{"id":20},"customfield_11502":"2014-05-09T10:38:53","customfield_11505":"Commit log:\n\n{\n    \"example\": \"manifest\"\n}"}}}