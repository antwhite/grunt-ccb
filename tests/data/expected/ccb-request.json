{"headers":{"content-type":"application/json","accept":"application/json","host":"localhost:8888","authorization":"Basic dXNlcjpwYXNzd29yZA==","content-length":"164","connection":"keep-alive"},"url":"/issue/","body":{"fields":{"project":{"id":"4321"},"summary":"Deploying project 0.0.1 to production","description":"Commit log:\n\n{\n    \"example\": \"manifest\"\n}","issuetype":{"id":20}}}}